README.md

```diff
@@ -6,15 +6,40 @@
 
 [![testing: coverage](https://codecov.io/gh/delvtech/agent0/branch/main/graph/badge.svg?token=1S60MD42ZP)](https://app.codecov.io/gh/delvtech/agent0?displayType=list)
 [![docs: build](https://readthedocs.org/projects/agent0/badge/?version=latest)](https://agent0.readthedocs.io/en/latest/?badge=latest)
-<br><a href="https://app.codecov.io/gh/delvtech/agent0?displayType=list"><img height="50px" src="https://codecov.io/gh/delvtech/agent0/graphs/sunburst.svg?token=1S60MD42ZP"><a>
 
 <img src="https://raw.githubusercontent.com/delvtech/agent0/main/icons/agent0-dark.svg" width="800" alt="agent0"><br>
+# Agent0 - A Framework for Hyperdrive Trading Strategies
 
-# [DELV](https://delv.tech) repo for market simulation and analysis
+This repo by [DELV](https://delv.tech) contain tools for you to deploy automated trading agents, perform market simulations, and conduct trading research. Read on for more info or jump [here](https://agent0.readthedocs.io/en/latest/#quickstart) to start.
```

Comment:
Avoid "click here/there/anywhere" style links in favor of in context links that don't disrupt natural reading.
```suggestion
This repo by [DELV](https://delv.tech) contain tools for you to deploy automated trading agents, perform market simulations, and conduct trading research. Read on for more info or jump into the [quickstart guide](https://agent0.readthedocs.io/en/latest/#quickstart).
```